### PR TITLE
Dockerfile for tests debian12

### PR DESCRIPTION
**Pull request by belamenso**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# `sbt update` is dependent on the build.sbt
+WORKDIR /app
+COPY build.sbt build.sbt
+RUN sbt update
+
+COPY src/ src/
+COPY project/build.properties properties/build.properties
+
+CMD ["sbt", "test"]
```
Comment (aherlihy) [inline Dockerfile:27]: The results of this should be stored or reported somewhere

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# tyql
+
+### Docker for running tests
+```bash
+$ docker build --network host -t tyql-test .
```
Comment (aherlihy) [inline README.md]: Should add that '--platform linux/x86_64' is needed for M1

**Review by aherlihy — APPROVED** [review]

**Comment by aherlihy** [comment]
How long does this take for you locally? It worked fine on the DIAS servers but locally it's been going for about 500s and still hasn't finished installing.

**Comment by belamenso** [comment]
This was replaced by something better

**Pull request closed without merging**